x-pack/plugins/secops/public/components/timeline/body/body.test.tsx

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { mount } from 'enzyme';
+import * as fp from 'lodash/fp';
```

Comment:
I have nothing against the `fp` import idiom, butt going to point out in `infra` and other projects they are selectively pulling functions out such as:

```ts
import get from 'lodash/fp/get';
import getOr from 'lodash/fp/getOr';
...
import { pipe } from 'lodash/fp';

```
So ... yea ... no changes requested just letting you know like the name fp, this will be a First Post! of sorts with using `fp`. But the documentation of lodash uses both `fp` and the individual atoms:

```
// Load the fp build.
var fp = require('lodash/fp');

// Load a method category.
var object = require('lodash/fp/object');
```
ref: https://github.com/lodash/lodash/wiki/FP-Guide

Maybe pulling out selective functions results in smaller builds?